### PR TITLE
Correct permissions on cassandra run script

### PR DESCRIPTION
**Pull request by uluyol**
Bash needs to be able to read the script to run it. Someone will need to push the docker image.

### CODE REVIEW

**Comment by uluyol** [comment]
Found another issue with this example, will push once fixed.

**Comment by uluyol** [comment]
Pushed.

File: examples/cassandra/README.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
-          image: gcr.io/google_containers/cassandra:v7
+          image: gcr.io/google_containers/cassandra:v8
```
Comment (brendandburns) [inline examples/cassandra/README.md]: We'll need to do this as a separate PR to ensure that we don't break users who read these docs before the image is pushed...

**Comment by brendandburns** [comment]
Thanks for the PR, LGTM, except the verison rev.  Once this PR is in, I'll build a new image and send a separate PR to rev the verison.

--brendan

**Comment by uluyol** [comment]
Pushed with version changes removed.
